### ISSUE TITLE
Continue writing to log file?

### ISSUE THREAD

**Issue by oystersauce8** [dwbutler/logstash-logger#52]
Is it possible to continue writing to the regular environment.log file, in the *default non-json rails log format*?

we use logstash logger like this
`
    config.logstash = [
      {
        type: :file,
        path: 'log/development.log'
      },
      {
        type: :file,
        path: '/var/log/development.log'
      }
    ]
`

**Comment by dwbutler** [comment]
Hi,

It's not possible to do this from a single Ruby `Logger` instance. A single `Logger` has a single `Formatter`.

I'm not saying it's impossible, but it's not something that LogStashLogger supports out of the box.

**Comment by oystersauce8** [comment]
i work next door to UCLA!   :)

**Comment by dwbutler** [comment]
That's cool! I graduated from UCLA in 2006 in physics. Now I live in Pasadena and work as a software engineer.

Back to the topic - The Ruby `Logger` class is designed to only log a single format. Internally, `LogStashLogger` only creates a single logger, that logs to multiple output devices. It can only log what is output by the formatter.

An alternative design we could explore would be to instead have multiple loggers, each with a configurable formatter.

**Comment by ffmike** [comment]
Alternatively you can use https://github.com/ffmike/multilogger as a shim in front of multiple logger instances - one from LogStashLogger and one from ActiveSupport.

**Comment by dwbutler** [comment]
@ffmike Thanks for the tip! I was planning to offer similar functionality in an upcoming release. Now that I've seen MultiLogger, I might just pull it in as a dependency instead of rolling my own.

**Comment by dwbutler** [comment]
@ffmike I was planning to pull in your code, but it doesn't appear to be released as a gem on rubygems.org. Mind if I vendor in your code and credit you?

**Comment by ffmike** [comment]
Sure, help yourself. I banged that together quickly for use in one of our internal projects and haven't bothered to gemify it.

**Comment by dwbutler** [comment]
@ffmike Thanks! I credited you in the readme.

Support for this released in 0.13.0. Example configuration:

```ruby
config.logstash.type = :multi_logger
config.logstash.outputs = [
  { type: :file, path: 'log/development.log', formatter: ::Logger::Formatter },
  { type: :file, path: '/var/log/development.log' }
] 
```
